src/components/v2/V2Project/V2PayButton/index.tsx

```diff
@@ -65,7 +69,13 @@ export default function V2PayButton({
           }}
           disabled={isPayDisabled}
         >
-          {payButtonText}
+          {fundingCycleLoading ? (
+            <div style={{ margin: '0 auto' }}>
+              <Loading size={'default'} />
```

Comment:
**suggestion** Think we should just use the `LoadingOutlined` from antd icons directly probs.

```ts
import { LoadingOutlined } from '@ant-design/icons'
```

Reply:
i rate it - i actually prefer it not colored too